Editor/HPUIStaticContinuousInteractableEditor.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using System.Linq;
+using ubco.ovilab.HPUI.StaticMesh;
+using ubco.ovilab.HPUI.Editor;
+using UnityEditor;
+using UnityEngine;
+
+namespace UnityLibrary
+{
+    [CanEditMultipleObjects]
+    [CustomEditor(typeof(HPUIStaticContinuousInteractable))]
+    public class HPUIStaticContinuousInteractableEditor : HPUIBaseInteractableEditor
+    {
+        private HPUIStaticContinuousInteractable t;
+        private SerializedProperty staticMesh;
+        private SerializedProperty meshXRes;
+        
+        protected override List<string> EventPropertyNames => base.EventPropertyNames.Union(new List<string>()
```

Comment:
Reset these values. Since you are manually drawing `staticMesh` and `meshXRes`, you may want them here instead of the values you have here. Also include "boundsCollider" as that's not relevant to the continuous HPUI.

Reply:
fixed 🫡